app.ts

```diff
@@ -31,6 +35,4 @@ app.use('/api', api_router)
 app.get('/', (req, res)=>{
```

Comment:
Two things here Server side rendering and graphQL are big in JS right now. I'd recommend next.js and apollo graphql

Reply:
I really like nextjs, it's so good!